components/atoms/FilterChip/filter-chip.tsx

```diff
@@ -0,0 +1,25 @@
+import clsx from "clsx";
+import React from "react";
+import { MdClose } from "react-icons/md";
+
+interface FilterChipProps {
+  className?: string;
+  items: string[];
+  onClear: () => void;
+}
+const FilterChip = ({ className, items, onClear }: FilterChipProps) => {
+  return (
+    <div
+      className={clsx(
+        "p-2.5 rounded-xl bg-sauced-light w-max text-sauced-orange max-w-[200px] flex items-center",
```

Comment:
`w-52` -> `208px`

Reply:
I'm using the `w-max` to resize the chip relative to its content and setting a limit to its growth with the `max-w-[200px]`. changing to `w-52` will alter the auto-resize of the chip. What I can do here is convert the `px` value to `rem`